build.gradle.kts

```diff
@@ -75,7 +79,10 @@ allprojects {
     }
   }
 
-  java { toolchain { languageVersion = JavaLanguageVersion.of(11) } }
+  java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+  }
```

Comment:
This `toolchain` thing fixes to a specific java version, which causes #237. Using source/target compatibility should work with any sdk version >= 17

Reply:
I tried now to enable the auto jvm download in the last commit, it sounds like a useful feature. I wonder though if it works on NixOS, perhaps @igalshilman give this a try to see if your machine likes it :)

Reply:
Amazing!

Reply:
I can confirm that it works on my machine as well. Thanks @slinkydeveloper for working your magic on this ❤️